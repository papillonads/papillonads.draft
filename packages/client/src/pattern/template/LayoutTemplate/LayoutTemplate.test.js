import React from 'react'
import layoutTemplateDefault from './LayoutTemplate.render'

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />,
  }
})

describe('<LayoutTemplate />', () => {
  describe('Snaphot', () => {
    test('must match layoutTemplateDefault', () => {
      expect(global.renderToJSON(layoutTemplateDefault)).toMatchSnapshot()
    })
  })
})
