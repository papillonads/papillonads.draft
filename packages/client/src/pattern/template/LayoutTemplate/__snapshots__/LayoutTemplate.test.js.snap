// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutTemplate /> Snaphot must match layoutTemplateDefault 1`] = `
<div>
  <h2 />
  <div>
    Under construction
     
    <span
      aria-label="emoji"
      role="img"
    >
      ðŸ™‚
    </span>
    <div />
  </div>
</div>
`;
